Help 146 memtier benchmark (#1382)
aiven/devportal: docs/products/redis/howto/benchmark-performance.rst

Benchmark performance
=====================

> Redis®* **memtier_benchmark** is a command line utility developed by Redis Labs (formerly Garantia Data Ltd.) for load generation and bechmarking NoSQL key-value databases. It is recommended for the performance benchmarking of Aiven for Redis®* services.

Reviewer: ```suggestion
Redis®* ``memtier_benchmark*`` is a command line utility developed by Redis Labs for load generation and performing benchmark NoSQL key-value databases. It is recommended for performing benchmark of Aiven for Redis®* services.
```

.. Warning::
    ``redis-benchmark`` is not supported to work with Aiven services, since ``CONFIG`` command is not allowed to run.
    
Prerequisites
-------------
* An Aiven for Redis service running.

> * ``memtier_benchmark`` installed. To install the tool, first download the source code from `Github <https://github.com/RedisLabs/memtier_benchmark>`_, and follow the instructions in the `README <https://github.com/RedisLabs/memtier_benchmark/blob/master/README.md>`_ to install all the dependencies, then build and install the tool. Now it's ready to go.

Reviewer: ```suggestion
* ``memtier_benchmark`` installed. To install the tool, first download the source code from `GitHub <https://github.com/RedisLabs/memtier_benchmark>`_, and follow the instructions in the `README <https://github.com/RedisLabs/memtier_benchmark/blob/master/README.md>`_ to install all the dependencies, then build and install the tool. Now it's ready to go.
```

Benchmark performance
=====================

Redis®* **memtier_benchmark** is a command line utility developed by Redis Labs (formerly Garantia Data Ltd.) for load generation and bechmarking NoSQL key-value databases. It is recommended for the performance benchmarking of Aiven Redis®* services.

.. Warning::
    ``redis-benchmark`` is not supported to work with Aiven services, since ``CONFIG`` command is not allowed to run.
    
Prerequisites
-------------
* An Aiven Redis service running.

> * **memtier_benchmark** installed. To install the tool, first download the source code from `github <https://github.com/RedisLabs/memtier_benchmark>`_, and follow the instructions in ``README`` to install all the dependencies, then build and install the tool. Now it's ready to go.

Reviewer: ```suggestion
* ``memtier_benchmark`` installed. To install the tool, first download the source code from `GitHub <https://github.com/RedisLabs/memtier_benchmark>`_, and follow the instructions in ``README`` to install all the dependencies, then build and install the tool. Now it's ready to go.
```

.. Note::
    The ``Testing`` section in ``README`` is not mandatory, can be skipped.

Running benchmark
-----------------

> Before using **memtier_benchmark**, read the help from :code:`mentier_benchmark -h` or this `Redis article <https://redis.com/blog/memtier_benchmark-a-high-throughput-benchmarking-tool-for-redis-memcached/>`_ to understand what the tool can offer.

Reviewer: ```suggestion
Before using ``memtier_benchmark``, read the help from :code:`mentier_benchmark -h` or this `Redis article <https://redis.com/blog/memtier_benchmark-a-high-throughput-benchmarking-tool-for-redis-memcached/>`_ to understand what the tool can offer.
```
